list.json

```diff
@@ -870,5 +870,14 @@
   "zgrab",
   "Zippy",
   "ZnajdzFoto/Image",
-  "ZyBorg"
+  "ZyBorg",
+  "WhatsApp",
+  "Twitterbot/1.0",
+  "LinkedInBot",
+  "Quora",
+  "Baidu",
+  "Applebot 0.1",
+  "TelegramBot",
+  "Slack",
```

Comment:
Could you specify the user agent string in the tests? Slack link checker (`Slackbot-LinkExpanding 1.0 (+https://api.slack.com/robots)`) is already covered by entry "bot"
https://github.com/gorangajic/isbot/blob/master/list.json#L122